cmd/fyne/internal/commands/build.go

```diff
@@ -55,12 +58,32 @@ func (b *builder) build() error {
 	if goos != "ios" && goos != "android" && goos != "wasm" {
 		env = append(env, "GOOS="+goos)
 	} else if goos == "wasm" {
+		versionConstraint = version.NewConstrainGroupFromString(">=1.17")
 		env = append(env, "GOARCH=wasm")
 		env = append(env, "GOOS=js")
 	}
 
 	cmd.Env = env
 
+	if versionConstraint != nil {
```

Comment:
Could this be factored out into a method, and a couple of validation tests added so we can check the parse/version comparison?

Reply:
I am trying to figure out how to write a test for this, but as it depends first on the output of `go version`, I am not sure how to proceed. Any suggestion?

Reply:
I would guess capturing certain known (or erroneous) outputs of `go version` and wire them into the test.
So the function would verify based on string not on a call out to a command.

Reply:
Hmm, what I meant is that we can feed in captured versions of the "go version" tool output and test using that. It seems reasonable to assume that to go tool will work, if present.